### PR TITLE
New package: roswell

### PR DESCRIPTION
**Pull request by LiamHowley**
While Roswell is not required by any other packages in the repository, it is fast becoming a required tool for developing in common lisp.

### CODE REVIEW

File: srcpkgs/roswell/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+license="MIT"
+homepage="https://github.com/roswell/roswell"
+distfiles="https://github.com/roswell/roswell/releases/download/v17.7.9.80/roswell_$version.orig.tar.gz"
+checksum=20e5a99a5865a906874c5361edd79e80445e8cb62d633aa55b77454eb798077a
+
+pre_configure() {
+	sh bootstrap
+}
+
+post_configure() {
```
Comment (the-maldridge) [inline srcpkgs/roswell/template]: This should be post_install

**Comment by LiamHowley** [comment]
Done - Thanks

**Comment by the-maldridge** [comment]
Changes look ok, just fix the commit message and it looks good to merge.

**Comment by LiamHowley** [comment]
Thanks for the pointers the-maldridge. That's done there now.

**Pull request closed without merging**